sky/packages/sky/lib/src/fn3/dialog.dart

```diff
@@ -148,8 +148,9 @@ class DialogRoute extends Route {
     );
   }
 
-  void popState([dynamic result]) {
+  void popped([dynamic result]) {
```

Comment:
didPop?  We're not all that consistent about how we name these notification methods.